cmd/gaia/app/genesis.go

```diff
@@ -27,8 +27,8 @@ import (
 
 var (
 	// bonded tokens given to genesis validators/accounts
-	freeFermionsAcc  = staking.TokensFromTendermintPower(150)
-	defaultBondDenom = staking.DefaultBondDenom
+	freeFermionsAcc  = sdk.TokensFromTendermintPower(150)
```

Comment:
`fermions`? This is a denom I haven't heard of.

Reply:
this is an antiquated denom - I'll rename